<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="TerraSanity"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="14.059">
  <properties />
            <testcase name="LoginIntoTerraEventApp on [CheckUserLogin passed=0 failed=0 skipped=0]" time="14.059" classname="testScripts.CheckUserLogin">
                <failure type="java.lang.AssertionError"
                  message="Actual and Expected Home page title is not matched"
              >
<![CDATA[java.lang.AssertionError: Actual and Expected Home page title is not matched
  org.testng.Assert.fail(Assert.java:99)
  pages.TerraLogin.LoginIntoBridgestone(TerraLogin.java:60)
  testScripts.CheckUserLogin.LoginIntoTerraEventApp(CheckUserLogin.java:19)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:564)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
  org.testng.TestRunner.privateRun(TestRunner.java:764)
  org.testng.TestRunner.run(TestRunner.java:585)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
  org.testng.SuiteRunner.run(SuiteRunner.java:286)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.runSuites(TestNG.java:1069)
  org.testng.TestNG.run(TestNG.java:1037)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
